libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -84,7 +84,12 @@ class AppPrompt {
   };
 
   doesEntitlementMatch = async () => {
-    const entitlements = await getConfig().entitlements();
+    const config = getConfig();
+    const entitlements = await config.entitlements();
+    if (config?.env?.name !== 'prod') {
+      const extraEnts = new URLSearchParams(window.location.search).get('mockPepEnts');
+      extraEnts?.split(',').forEach((ent) => entitlements.push(ent.trim()));
+    }
```

Comment:
Is content QA not required on prod as well? From what I heard, it's always a pain to set this up and consequently should work on every environment. Since this just runs if the query parameter is available IMO there is no harm to also run this on prod.

Rather than trying to get the right entitlements, should we just make this a bool?

E.g. `skipPepEntitlements=true`

L53/54 in the init:
```js
    const skipEntitlements = new URLSearchParams(window.location.search).get('skipPepEntitlements');
    const entMatch = skipEntitlements || await this.doesEntitlementMatch();
    if (!entMatch) return;
```

Reply:
Enabling this on prod is fine by me. It might be worth having both the ability to add specific entitlements, and to skip the whole thing altogether.

I'll run these by MarkL and add them in.

Reply:
> Enabling this on prod is fine by me. It might be worth having both the ability to add specific entitlements, and to skip the whole thing altogether.
> 
> I'll run these by MarkL and add them in.

Mark is fine with the additions and I've added them in. So this should be good to merge.